### PR TITLE
Pipeline: Update NumPy array cropping condition

### PR DESCRIPTION
**Pull request by UdomkarnBoonyaprasert**
Cropping is now done if any of the dimensions is not divisible by 8. This is to address issue #56

### CODE REVIEW

File: HoloPipelines/core/services/np_image_manipulation.py [hunk 1]
```diff
@@ -51,11 +51,11 @@ def downscale_and_conditionally_crop(
         x = image.shape[0]
```
Comment (nielsboecker) [inline HoloPipelines/core/services/np_image_manipulation.py]: The FIXME comment in line 13 should be removed if you fixed it

Comment (nielsboecker) [reply HoloPipelines/core/services/np_image_manipulation.py]: crop_around_centre should have a doc string and the arguments should have better names

Comment (UdomkarnBoonyaprasert) [reply HoloPipelines/core/services/np_image_manipulation.py]: FIXME comment removed and arguments naming updated

**Comment by nielsboecker** [comment]
@UdomkarnBoonyaprasert Code looks great now 👍. How about adding a test before we close the issue? To assert the different scenarios work:
```
np_image_manipulation_test.py
    test_crop_around_centre
    test_downscale_and_conditionally_crop_with_only_downscale
    test_downscale_and_conditionally_crop_with_downscale_and_crop
    test_downscale_and_conditionally_crop_with_only_crop
```

**Review by nielsboecker — APPROVED** [review]
Thanks @UdomkarnBoonyaprasert, nice testing!
FYI please take a look at this small change I made: 915b319
I will merge now, and this resolves #56

**Pull request merged**